blocks/color-palette/index.js

```diff
@@ -61,15 +61,11 @@ function ColorPalette( { colors, value, onChange } ) {
 				) }
 			/>
 
-			<div className="blocks-color-palette__item-wrapper blocks-color-palette__clear-color">
-				<button
-					className="blocks-color-palette__item"
-					onClick={ () => onChange( undefined ) }
-					aria-label={ __( 'Remove color' ) }
-				>
-					<span className="blocks-color-palette__clear-color-line" />
-				</button>
-			</div>
+			<button className="button-link blocks-color-palette__clear" type="button"
```

Comment:
I'd suggest one of two conventions for applying props to an element:

Single line

```jsx
<div className="..." onClick={ () => {} }>
	Hello World
</div>
```

Multiple lines (particularly when single line exceeds 80 characters):

```jsx
<div
	className="..."
	onClick={ () => {} }
>
	Hello World
</div>
```

Currently, this is an awkward mix of single- and multi-line style.

Reply:
Corrected 👍